content/CORE/Solutions/Integrations/Nextcloud.md

```diff
@@ -25,105 +21,80 @@ To see the plugin catalog, go to the **Plugins** screen.
 {{< /expand >}}
 
 ![PluginsList](/images/CORE/12.0/PluginsList.png "Plugins Catalog")
-{{< expand "I don't see anything?" "v" >}}
+{{< expand "I don't see anything!" "v" >}}
 If the catalog doesn't load:
 * Go to **Network > Global Configuration** and confirm the **Default Gateway** and **DNS Servers** addresses are correct.
-* Open the **Shell** and `ping` an Internet address. The output confirms the system is connected to the Internet.
+* Open the **Shell** and `ping` an Internet address. 
+The output confirms the system is connected to the Internet.
 {{< /expand >}}
 
-Plugins are organized into two **Collections**:
+TrueNAS organizes plugins into two **Collections**:
 
 * [iXsystems](https://www.ixsystems.com/) maintained plugins
-* Open Source plugins created and maintained by the TrueNAS community.
+* Open Source plugins created and maintained by TrueNAS users.
 
-By default, the iXsystems-supported plugins are shown.
-To view the community-supported plugins, open *Browse a Collection* and select *Community*.
+By default, TrueNAS shows the iXsystems-supported plugins.
+To see the community plugins, open **Browse a Collection** and select **Community**.
 
-### Basic Install 
+### Installation
 
-Go to **Plugins** select *Nextcloud* and click the *INSTALL* button.
+{{< tabs "Install Types" >}}
+{{< tab "Basic Install" >}}
+Go to **Plugins** and select **Nextcloud**, then click **INSTALL**.
 
 ![NextcloudInstall](/images/CORE/12.0/SolutionsIntegrationsNextcloudInstall.png "Nextcloud Install")
 
-For *Plugins / Add* choose a *Jail Name* and click *SAVE*
+Type a **Jail Name** and click **SAVE**.
 
 ![NextcloudJailName](/images/CORE/12.0/SolutionsIntegrationsNextcloudJailName.png "Nextcloud Jail Name")
 
-Once Nextcloud has installed successfully, you can then begin to manage your instance of the plugin.  
+Once Nextcloud installs successfully, you can manage your instance of the plugin.  
 
 ![NextcloudPostInstall](/images/CORE/12.0/SolutionsIntegrationsNextcloudPostInstall.png "Nextcloud Post Install")
 
-Select *POST INSTALL NOTES* to obtain your *Nextcloud Admin User* and *Nextcloud Admin Password* information.  Select *MANAGE* and you will be directed to the Nextcloud login page within your browser.
+Click **POST INSTALL NOTES** to obtain your *Nextcloud Admin User* and *Nextcloud Admin Password* information. 
```

Comment:
if you are specifically talking about the UI elements as UI elements they are bold, if only talking about them in general  as an item or value for the UI fields then no emphasis is necessary. 
```suggestion
Click **POST INSTALL NOTES** to obtain your Nextcloud admin user and  Nextcloud admin password information. 
```